app/build.gradle

```diff
@@ -168,10 +175,16 @@ android {
         def needed = variant.name in [
                 'googlevrArmDebug',
                 'googlevrArmRelease',
+                'googlevrArm64Debug',
+                'googlevrArm64Release',
                 'oculusvrArmDebug',
                 'oculusvrArmRelease',
+                'oculusvrArm64Debug',
+                'oculusvrArm64Release',
                 'svrArmDebug',
                 'svrArmRelease',
+                'svrArm64Debug',
+                'svrArm64Release',
                 'wavevrArmDebug',
                 'wavevrArmRelease',
                 'noapiArmDebug',
```

Comment:
Can you add noapiArm64[Debug,Release] also?

Reply:
Done